pkg/controller/common/manifestprocessing.go

```diff
@@ -43,28 +43,29 @@ func NewManifestProcessor(controllerResources ControllerResources, appInstance,
 func (p *ManifestProcessor) ProcessManifests(ctx context.Context, manifests []manifest.Manifest, component string) error {
 	allErrors := []error{}
 
-	origLogger := p.Log
-	defer func() { p.Log = origLogger }()
+	origCtx := ctx
+	origLogger := LogFromContext(ctx)
```

Comment:
Would it be better to create a new child context, which includes the new logger?

Reply:
Fixed in #324